doc/src/manual/types.md

```diff
@@ -634,20 +631,16 @@ Point{Float64}
 
 For the default constructor, exactly one argument must be supplied for each field:
 
-```julia
+```jldoctest pointtype
 julia> Point{Float64}(1.0)
 ERROR: MethodError: Cannot `convert` an object of type Float64 to an object of type Point{Float64}
 This may have arisen from a call to the constructor Point{Float64}(...),
 since type constructors fall back to convert methods.
- in Point{Float64}(::Float64) at ./sysimg.jl:66
- ...
+Stacktrace:
+ [1] Point{Float64}(::Float64) at ./sysimg.jl:24
 
 julia> Point{Float64}(1.0,2.0,3.0)
 ERROR: MethodError: no method matching Point{Float64}(::Float64, ::Float64, ::Float64)
-Closest candidates are:
-  Point{Float64}{T}(::Any, ::Any) at none:3
-  Point{Float64}{T}(::Any) at sysimg.jl:66
```

Comment:
bisected down to one of
3305d93fc29b9e326702a1928ebf4036beaa09cb
b32dd7c1a65cc6e2a96c850d26f5d88e26f1664c
e3bec2a34d724843040e8101d22c5bfacf891968
that made this stop printing - worth an issue I think